images/cilium/Dockerfile

```diff
@@ -1,47 +1,90 @@
-# syntax=docker/dockerfile:1.1-experimental
+# syntax=docker/dockerfile:1.2
 
 # Copyright 2020-2021 Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-ARG CILIUM_BUILDER_IMAGE=docker.io/cilium/cilium-builder-dev:879b19d3b784fdd0dda9f91310107c6fe209ffac
-ARG CILIUM_RUNTIME_IMAGE=docker.io/cilium/cilium-runtime-dev:0609ab6cc05c0dd78499c417c9ce3d0dc24c7f54
+# cilium-envoy from github.com/cilium/proxy
+#
+FROM quay.io/cilium/cilium-envoy:1bfbbd9c85907c5667f1abd24b7a6c1b5d8bbf29@sha256:05f556d53b53b062c5ec09fb153f0d0f06b470bc05b025e965bc36f3fb307b6d as cilium-envoy
 
-FROM ${CILIUM_BUILDER_IMAGE} as builder
+#
+# Hubble CLI
+#
+# FIXME add multi-arch support for cilium/hubble!
+FROM --platform=${BUILDPLATFORM} quay.io/cilium/hubble:v0.7.1@sha256:cc76aa6394d613eaeeac0f15b72f50d426b3c47d4676557431661e6aa5e1597b as hubble
```

Comment:
We probably could switch over to downloading the CLI from the GitHub releases instead of extracting the binary from the image. They are built for various architectures:

https://github.com/cilium/hubble/releases/tag/v0.7.1